src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,20 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        enum ColorBase{
+            BLUE,
+            YELLOW,
+            GREEN,
+            RED;
+        }
+
+        Random random = new Random();
```

Comment:
The `Random` instance `random` can be a private final class-level field instead of being created inside the `getRandomColor` method. This way, you avoid creating a new `Random` object on each method call, which is more efficient.